package.json

```diff
@@ -36,6 +36,7 @@
     "selenium-webdriver": "2.45.1"
   },
   "devDependencies": {
+    "angular-history": "git+https://github.com/decipherinc/angular-history.git#v0.8.0",
```

Comment:
Based on this logic, if we consider our scenario where the main repo `superdesk` embeds this one as a module, then all of the `devDependencies` should be in `dependencies`. I am also not sure of the implications of that. But at least it would remove the duplication of `package.json`